ankihub/main/review_data.py

```diff
@@ -146,3 +148,18 @@ def get_daily_review_summaries_since_last_sync(
             )
         )
     return daily_card_review_data
+
+
+def send_daily_review_summaries(last_summary_sent_date: date) -> None:
+    """Send daily review summaries to the server."""
+    daily_review_summaries = get_daily_review_summaries_since_last_sync(
```

Comment:
:seedling: If a user doesn't use AnkiHub for a long time and then starts using it again, there can be a lot of daily summaries to send. 
Each summary is sent using its own request, so if they had a pause of 150 days, there would potentially be 150 requests...
It would be good to have an endpoint which accepts a list of summaries.

Reply:
Good point, I will create these tasks as a nice to have (feel free to also create these tasks)